node-src/tasks/gitInfo.ts

```diff
@@ -260,6 +269,13 @@ export const setGitInfo = async (ctx: Context, task: Task) => {
     }
   }
 
+  ctx.projectMetadata = {
+    hasRouter: getHasRouter(ctx.packageJson),
+    creationDate: await getRepositoryCreationDate(),
+    numberOfCommitters: await getNumberOfComitters(),
+    numberOfAppFiles: await getCommittedFileCount(['page', 'screen'], ['js', 'jsx', 'ts', 'tsx']),
+  };
```

Comment:
There are cases where TurboSnap returns early. Should we move this up so we aren't missing this metadata when that happens?